examples/trl/requirements.txt

```diff
@@ -0,0 +1,10 @@
+torch --index-url https://download.pytorch.org/whl/cu124
+torchvision --index-url https://download.pytorch.org/whl/cu124
```

Comment:
probably torchvision & torchaudio can be removed here .. also I would remove the special version 12.4 and add used  `>= 2.6.0` instead (cuda toolkit 12.4 is currently default).